server/graphql/asset/src/logs/insert.rs

```diff
@@ -22,7 +22,7 @@ pub fn insert_asset_log(
     let user = validate_auth(
         ctx,
         &ResourceAccessRequest {
-            resource: Resource::MutateAssetLog,
+            resource: Resource::QueryAsset,
```

Comment:
we could - and I was tempted too because that's nice. 
Richard said that everyone should be able to update the status though, so that's what I've done